lib/src/bindings/4.0/generated_bindings.dart

```diff
@@ -2759,9 +2759,9 @@ class Tizen40Native {
   }
 
   late final _notification_set_time_to_textPtr = _lookup<
-      ffi.NativeFunction<
-          ffi.Int Function(notification_h, ffi.Int32,
-              time_t)>>('notification_set_time_to_text');
+          ffi
+          .NativeFunction<ffi.Int Function(notification_h, ffi.Int32, time_t)>>(
+      'notification_set_time_to_text');
```

Comment:
Are these only formatting changes?

Reply:
The format has changed. In my view, there doesn't seem to be any API change.

Reply:
We didn't update the ffigen version. What made the changes?

→ ffigen 7.2.10 added parameter names for function pointer arguments.

> 7.2.10 [#](https://pub.dev/packages/ffigen/changelog#7210) 
> Generate parameter names in function pointer fields and typedefs.

You can update the dev_dependency in `pubspec.yaml` to the version installed on your machine if you want to be clear.

Reply:
Thank you for your review. 
I updated ffigen version to latest(11.0.0). And I reverted the ffigen.yaml(4.0 ~ 7.0) changes.
 +) I updated the version of the tizen_interop_callbacks package to 0.1.1.
please let me know if there's more to add.

Reply:
Tip: Please check out the changelog and README of the ffigen package for newly added, potentially useful configuration options and other features when updating the ffigen version.

Reply:
As far as I know, the minimum version we need from ffigen is 8.0.0-dev.0.
( 8.0.0-dev.0 : Adds final class modifier to generated sub types Struct, Union and Opaque...)
If we use ffigen 8.0.0, we need to adjust `file` package version, but `file` package applies the fixes made from dart 3.0 in 7.0.0.
Therefore, in order to satisfy the version, ffigen version should be 11.0.0.